nextflow.config

```
@@ -29,20 +25,19 @@ params {
     multiqc_methods_description = null
 
     // Boilerplate options
-    outdir                       = null
+    outdir                       = "null"
     publish_dir_mode             = 'copy'
     email                        = null
     email_on_fail                = null
     plaintext_email              = false
     monochrome_logs              = false
-    hook_url                     = System.getenv('HOOK_URL')
+    hook_url                     = null
```

```suggestion
    hook_url                     = System.getenv('HOOK_URL')
```
This should be reverted